lib/iris/tests/test_coord_api.py

```diff
@@ -178,7 +178,9 @@ def test_complex(self):
 @tests.skip_data
 class TestCoord_ReprStr_nontime(tests.IrisTest):
     def setUp(self):
-        self.lat = iris.tests.stock.realistic_4d().coord("grid_latitude")[:10]
+        cube = iris.tests.stock.realistic_4d()
+        self.lat = cube.coord("grid_latitude")[:10]
+        self.height = cube.coord("level_height")[:10]
```

Comment:
These tests were a bit of a mess : these 4 things were 2 duplicates of only 2 things.
Unfortunately I'm not sure I've improved things much, though.
'height' is not actually an AuxCoord (class), though it is an aux-coord in the cube.